symengine/cwrapper.cpp

```diff
@@ -160,7 +160,7 @@ TypeID basic_get_type(const basic s)
     return static_cast<TypeID>(s->m->get_type_code());
 }
 
-void symbol_set(basic s, char *c)
+void symbol_set(basic s, const char *c)
```

Comment:
I was under that impression that I can't pass a `const <type>` to a function that takes `<type>` which is why I sent this PR. Turns out I was wrong and I can indeed do it.